rllib/algorithms/a2c/tests/test_a2c.py

```diff
@@ -13,15 +14,15 @@ class TestA2C(unittest.TestCase):
     """Sanity tests for A2C exec impl."""
 
     def setUp(self):
-        ray.init(num_cpus=4)
+        ray.init()
 
     def tearDown(self):
         ray.shutdown()
 
     def test_a2c_compilation(self):
         """Test whether an A2C can be built with both frameworks."""
         config = a2c.A2CConfig().rollouts(num_rollout_workers=2, num_envs_per_worker=2)
-
+        config = add_gpu_if_necessary(config)
```

Comment:
actually, wouldn't
```
config = a2c.A2CConfig()\
    .rollouts(num_rollout_workers=2, num_envs_per_worker=2)\
    .resources(num_gpus=os.environ.get("RLLIB_NUM_GPUS"))
```
be more expressive? If `RLLIB_NUM_GPUS=None` (not defined), `.resources()` wouldn't do anything (not change the default value).

Hmm, matter of taste, I guess; feel free to leave the utility function in place.